nuxeo-services/login/nuxeo-platform-login-keycloak/src/main/java/org/nuxeo/ecm/platform/ui/web/keycloak/KeycloakAuthenticationPlugin.java

```diff
@@ -173,7 +173,11 @@ private KeycloakUserInfo getKeycloakUserInfo(AccessToken token) {
      */
     private Set<String> getRoles(AccessToken token, String keycloakNuxeoApp) {
         Set<String> allRoles = new HashSet<>();
-        Set<String> roles = token.getRealmAccess().getRoles();
+        Set<String> roles = null;
+        if(token.getRealmAccess()!=null) {
```

Comment:
Not well formatted, see https://doc.nuxeo.com/corg/java-code-style/

Reply:
It has been corrected